exonum/CHANGELOG.md

```diff
@@ -34,6 +34,15 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   the `execute` method. More generally, make sure that the method returns `Ok(())`
   on successful execution. (#385)
 
+- Service transactions are now defined through `transactions!` macro. (#457)
+
+  Migration path:
+
+  - Move all separate transactions declared as `message!` into one
+  `transactions!` macro.
+  - Remove `ID` constants.
```

Comment:
Let’s mention that IDs are assigned automatically based on the declaration order: the client needs to know ID to create a transaction.

Reply:
Fixed.